internal/locate/region_request.go

```diff
@@ -573,18 +573,22 @@ func (state *accessFollower) next(bo *retry.Backoffer, selector *replicaSelector
 	if state.option.preferLeader {
 		state.lastIdx = state.leaderIdx
 	}
+	offset := rand.Intn(replicaSize)
```

Comment:
Under `PreferLeader` mode, offset should be set to `0` to make the selection starting from `leaderIdx`.

Reply:
Yeah, I changed the code and not to use offset in the first selection.

Reply:
Recommend move `offset` before the judgement of `if state.option.preferLeader {`.